### PR TITLE
[ADVAPP-706]: Create report for the report library for Prospects

### PR DESCRIPTION
**Pull request by jayushi-canyon**
### Ticket(s) or GitHub Issue

- https://canyongbs.atlassian.net/browse/ADVAPP-706

### Technical Description

> Create report in report library for Prospects module

### Any deployment steps required?

> No.

### Are any Feature Flags Added?

> No.

_______________________________________________

### CODE REVIEW

**Comment by jayushi-canyon** [comment]
@Orrison Created this new PR for ADVAPP-706, As there were some issue while taking latest pull from main in that branch. I will delete the previous PR.

File: app-modules/report/src/Filament/Widgets/ProspectReportStats.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                maxPrecision: 2,
+            )),
+            Stat::make('Total Alerts', Number::abbreviate(
+                Cache::tags([$this->cacheTag])->remember('prospect-alerts-count', now()->addHours(24), function (): int {
+                    return Alert::whereHasMorph('concern', Prospect::class)->count();
+                }),
+                maxPrecision: 2,
+            )),
+            Stat::make('Total Segments', Number::abbreviate(
+                Cache::tags([$this->cacheTag])->remember('pospect-segments-count', now()->addHours(24), function (): int {
```
Comment (Orrison) [inline app-modules/report/src/Filament/Widgets/ProspectReportStats.php:72]:
```suggestion
                Cache::tags([$this->cacheTag])->remember('prospect-segments-count', now()->addHours(24), function (): int {
```

**Review by Orrison — CHANGES_REQUESTED** [review]
Looks good. Just one small typo.

File: app-modules/report/src/Filament/Widgets/ProspectReportStats.php (hunk 1 above)
Comment (jayushi-canyon) [reply app-modules/report/src/Filament/Widgets/ProspectReportStats.php]: @Orrison Done

**Review by Orrison — APPROVED** [review]

**Pull request merged**